Source/WTF/wtf/ThreadSafeWeakPtr.h

```diff
@@ -43,11 +43,8 @@ class ThreadSafeWeakPtrControlBlock {
     ThreadSafeWeakPtrControlBlock* weakRef()
     {
         Locker locker { m_lock };
-        if (m_object) {
-            ++m_weakReferenceCount;
-            return this;
-        }
-        return nullptr;
+        ++m_weakReferenceCount;
```

Comment:
ASSERT(m_object) here since we are removing the null check?  Or keep null check?

Reply:
I think folks pass around a dead ControlBlock but I can try adding the ASSERT and see if there are crashes.